### PR TITLE
Feat: make ping configurable

### PR DESCRIPTION
**Pull request by zhouaihui**
About #91 
It's better not to ping others by default.

### CODE REVIEW

**Review by ian-huu — APPROVED** [review]
LGTM

**Comment by miluoalbert** [comment]
LGTM

File: fed/api.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def init(
     cross_silo_serializing_allowed_list: Dict = None,
     exit_on_failure_cross_silo_sending: bool = False,
     cross_silo_messages_max_size_in_bytes: int = None,
+    enable_cross_silo_ping: bool = False,
```
Comment (jovany-wang) [inline fed/api.py:58]:
```suggestion
    enable_waiting_for_other_parties_ready: bool = False,
```
FYI: Just asked chatgpt for a good variable name~~~

**Review by jovany-wang — APPROVED** [review]
Just suggested a better variable name.

File: fed/api.py (hunk 1 above)
Comment (zhouaihui) [reply fed/api.py]: Nice suggestion!

**Pull request merged**